src/nim_status_client.nim

```diff
@@ -112,6 +112,7 @@ proc mainProc() =
   let appController = newAppController(statusFoundation)
   let networkAccessFactory = newQNetworkAccessManagerFactory(TMPDIR & "netcache")
 
+  let isDevelopmentQVariant = newQVariant(if defined(development): true else: false)
   let isProductionQVariant = newQVariant(if defined(production): true else: false)
```

Comment:
Can we just reuse `production` instead? I'm pretty sure they are and will stay mutually exclusive.

Reply:
Yeah, you're right, will simplify this

Reply:
@jrainville done